src/components/editor-page/editor-page.tsx

```diff
@@ -55,7 +58,7 @@ export const EditorPage: React.FC = () => {
       if (scrollSource.current === ScrollSource.RENDERER && editorSyncScroll) {
         setScrollState((old) => {
           const newState = { editorScrollState: newScrollState, rendererScrollState: old.rendererScrollState }
-          console.debug('[EditorPage] set scroll state because of renderer scroll', newState)
+          log.debug('[EditorPage] set scroll state because of renderer scroll', newState)
```

Comment:
We don't need the prefix here.

Reply:
Strange. I could swear I already removed this o.ö